apps/camera/js/controllers/preview-gallery.js

```diff
@@ -77,6 +77,11 @@ PreviewGalleryController.prototype.openPreview = function() {
   this.view.on('loadingvideo', this.app.firer('busy'));
   this.view.on('playingvideo', this.app.firer('ready'));
 
+  // Screen reader right/left swipes with two fingers to navigate to
+  // next/previous frames.
+  this.view.on('wheel:right', this.next);
+  this.view.on('wheel:left', this.previous);
```

Comment:
We should have the view listen for the `wheel` DOM event from the screen reader and have the view emit a `swipe` event that this controller listens for. This controller is already handling `swipe` events that take a `direction` argument which is either 'left' or 'right'.